editing/run/META.yml

```diff
@@ -0,0 +1,17 @@
+links:
+    - product: chrome
+      url: https://crbug.com/1002937
+      results:
+        - test: caret-navigation-around-line-break.html
+    - product: chrome
+      url: https://crbug.com/1168974
+      results:
+        - test: delete.html?1-1000
```

Comment:
None of the failures in this subset seem to be about tables. Can you trim this list to just the ones listed in the bug description?

Reply:
Done.